d3a_api_client/aggregator.py

```diff
@@ -39,10 +39,11 @@ def received_message(self, message):
 class Aggregator(RestDeviceClient):
 
     def __init__(self, simulation_id, domain_name, aggregator_name,
-                 websockets_domain_name, accept_all_devices=True):
+                 websockets_domain_name, autoregister=False, accept_all_devices=True):
```

Comment:
the aggregator does not register. This is why Spyros left this keyword out

Reply:
Remnant of testing, will remove it